static/js/settings_bots.js

```diff
@@ -83,6 +83,53 @@ exports.generate_flaskbotrc_content = function (email, api_key) {
            "\n";
 };
 
+exports.bots_allowed_for_non_admins_values = {
+    everyone: {
+        code: 1,
+        description: "Everyone",
```

Comment:
These strings, like anything else user-visible in the UI, must be tagged for translation (in this case, with `i18n.t()`).  And because `i18n.t()` isn't available at module load time, you'll need to move this to e.g. the `set_up` method.  See for example `_setup_page` in `static/js/settings.js` for how we do this sort of thing.

Reply:
Moving it to `settings_bots.set_up` raises many errors because `update_bot_settings_tip` uses these values and it is called in admin.js. The solution here is to move these values to admin.js where it works fine. Is this alright?

Reply:
So, I came up with this easy solution:
in `settings_bots.js`-
```
exports.bot_creation_policy_values = {};

exports.setup_bot_creation_policy_values = function () {
    if (exports.bot_creation_policy_values === undefined) {
        exports.bot_creation_policy_values = {
            everyone: {
                code: 1,
                description: i18n.t("Everyone"),
            },
            admins_only: {
                code: 3,
                description: i18n.t("Admins only"),
            },
            restricted: {
                code: 2,
                description: i18n.t("Everyone, but only admins can add generic bots"),
            },
        };
    }
};
```
And since settings.setup is called first so, I just called above function in there:
```
    settings_bots.setup_bot_creation_policy_values();
```

Reply:
Looks great, but you should skip the `if` statement, since it's both wrong (`{} !== undefined`) and unnecessary; the 3 dict lookups (which is what `i18n.t()` does) is so fast that there's no real benefit in writing extra potentially buggy code to avoid doing it twice.